tests/test.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+import os
```

Comment:
it solves issue of importError. without the lines i get the ModuleNotFoundError.